SwiftVoiceCallKitQuickstart/ViewController.swift

```diff
@@ -241,19 +241,20 @@ class ViewController: UIViewController, PKPushRegistryDelegate, TVONotificationD
         reportIncomingCall(from: "Voice Bot", uuid: callInvite.uuid)
     }
     
-    func handleCallInviteCanceled(_ callInvite: TVOCallInvite) {
+    func handleCallInviteCancelled(_ cancelledCallInvite: TVOCancelledCallInvite) {
         NSLog("callInviteCanceled:")
         
-        performEndCallAction(uuid: callInvite.uuid)
+        if (self.callInvite == nil ||
+            self.callInvite!.callSid != cancelledCallInvite.callSid) {
```

Comment:
why is it required for `callInvite!.callSid`?

Reply:
Checking the Call SID just in case the received cancellation isn't for the invite being processed. In the CallKit sample I think some of the developers are actually running into such issue - for some reason the cancelled invite received of the previous call is received and the app just proceed and tell CallKit to end the call, with an UUID that does not exist thus results in error.